cloud/firestore/v1/document_event_data.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package firestore
```

Comment:
Only comment, basically: should this package match the directory structure? (Should it be cloud/firestore/v1 or similar?)

Reply:
Go is a little different, and packages are named quite differently and don't have a fully qualified name.

Modeled after https://github.com/googleapis/google-api-go-client

Will definitely check with a Go expert.